docs/virtual_machines/net_binding_plugins/passt.md

```diff
@@ -0,0 +1,211 @@
+# Passt binding
+
+## Overview
+
+[Plug A Simple Socket Transport](https://passt.top/passt/about/) is an enhanced
+alternative to [SLIRP](https://en.wikipedia.org/wiki/Slirp), providing user-space
+network connectivity.
+
+`passt` is a universal tool which implements a translation layer between
+a Layer-2 network interface and native Layer -4 sockets (TCP, UDP, ICMP/ICMPv6 echo)
+on a host.
+
+Its main benefits are:
+
+- Doesn't require extra network capabilities as CAP_NET_RAW and CAP_NET_ADMIN.
+- Allows integration with service meshes (which expect applications to run locally) out of the box.
+- Supports IPv6 out of the box (in contrast to the existing bindings which require configuring IPv6
+  manually).
+
+### Functionality support
+| Functionality                                  | Support |
+|------------------------------------------------|---------|
+| Migration support                              | No      |
+| Service Mesh support                           | Yes     |
+| Pod IP in guest                                | Yes     |
+| Custom CIDR in guest                           | No      |
+| Require extra capabilities (on pod) to operate | No      |
+| Primary network (pod network)                  | Yes     |
+| Secondary network                              | No      |
+
+### Node optimization requirements/recommendations:
+
+1. To get better performance the node should be configured with:
+```
+sysctl -w net.core.rmem_max = 33554432
+sysctl -w net.core.wmem_max = 33554432
+```
+2. To run multiple passt VMs with no explicit ports, the node's `fs.file-max` should be increased
+   (for a VM forwards all IPv4 and IPv6 ports, for TCP and UDP, passt needs to create ~2^18 sockets):
+```
+sysctl -w fs.file-max = 9223372036854775807
+```
+
+> **NOTE**: To achieve optimal memory consumption with Passt binding,
+> specify ports required for your workload.
+> When no ports are explicitly specified, all ports are forwarded,
+> leading to memory overhead of up to 800 Mi.
+
+## Passt network binding plugin
+[v1.1.0]
+
+The binding plugin replaces the experimental core passt binding implementation
+(including its API).
+
+> **Note**: The network binding plugin infrastructure and the passt plugin
+> specifically are in Alpha stage. Please use them with care, preferably
+> on a non-production deployment.
+
+The passt binding plugin consists of the following components:
+
+- Passt CNI plugin.
+- Sidecar image.
+
+As described in the [definition & flow](#definition--flow) section,
```

Comment:
@EdDev What is this meant to link to? I can't seem to find a related heading in any of our docs

Reply:
It is here: https://github.com/kubevirt/user-guide/pull/742/files#diff-954edce67fbabb116f75540df31fbd8ceeb7de6a02cebd186d92880c33563a69R49

I'll send a fix, thanks.

Reply:
Done: https://github.com/kubevirt/user-guide/pull/793